specification/purview/data-plane/Azure.Analytics.Purview.Scanning/preview/2022-07-01-preview/scanningService.json

```
@@ -0,0 +1,9155 @@
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "managedVNetName",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "default"
+            ],
+            "x-ms-enum": {
+              "name": "ManagedVirtualNetworkName",
+              "modelAsString": true
+            }
+          },
+          {
+            "in": "path",
+            "name": "managedPEName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ManagedPrivateEndpoint"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Scanning Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ManagedVirtualNetworks_GetPE": {
+            "$ref": "./examples/ManagedVirtualNetworks_GetPE.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ManagedVirtualNetworks"
+        ],
+        "description": "Deletes a ManagedPrivateEndpoint",
+        "operationId": "ManagedVirtualNetworks_DeletePE",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "managedVNetName",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "default"
+            ],
+            "x-ms-enum": {
+              "name": "ManagedVirtualNetworkName",
+              "modelAsString": true
+            }
+          },
+          {
+            "in": "path",
+            "name": "managedPEName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/IActionResult"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Scanning Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ManagedVirtualNetworks_DeletePE": {
+            "$ref": "./examples/ManagedVirtualNetworks_DeletePE.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ManagedVirtualNetworks"
+        ],
+        "description": "Creates a ManagedPrivateEndpoint",
+        "operationId": "ManagedVirtualNetworks_CreateOrUpdatePE",
```

Put is create or replace
```suggestion
        "description": "Create or replace a ManagedPrivateEndpoint",
        "operationId": "ManagedVirtualNetworks_CreateOrReplacePE",
```

---

changed